docs/source/module_configuration/cloudformation.rst

```diff
@@ -1,108 +1,142 @@
 .. _CFNgin: ../cfngin/index.html
-.. _Lookups: lookups.html
+.. _CFNgin Config File: ../cfngin/config.html
+.. _deploy environment: ../terminology.rst#deploy-environment
+.. _environment file: ../cfngin/environments.html
+.. _Lookups: ../lookups.html
+.. _Parameters: ../terminology.html#parameters
+.. _Runway Config File: runway_config.html
 
 .. _mod-cfn:
 
 CloudFormation
 ==============
 
-CloudFormation modules are managed by 2 files:
+CloudFormation modules are processed using CFNgin.
+There are two files that can be used for configuration:
+
+- a YAML `CFNgin Config File`_
```

Comment:
For any of these touchpoints (places in the docs where initial/basic users of Runway are looking for info on defining stacks & params, etc), I'd really like to demphasize the CFNgin name. Specifically, it'd be nice if we could avoid having people ask questions about runway/cfngin, and just have them ask about "runway".

So for this specific place, something like a "YAML Runway CloudFormation Config File" I _think_ would be better for that goal.

Reply:
makes sense - just a bit of a mouth full. I'll go back through a reword this page tomorrow (or later tonight if I get a chance).